source/PlayState.hx

```diff
@@ -168,13 +173,19 @@ class PlayState extends FlxState
 		FlxG.overlap(player, dtmEnemy1, dtmEnemy1.playerHitResolve);
 		FlxG.overlap(player, bullets, bulletHitPlayer);
 		FlxG.overlap(player, trap._grpBarTrap, trap.playerTrapResolve);
+		FlxG.overlap(player, sword, player.pickup_item);
+		FlxG.overlap(sword.hitbox.hitboxFrames, dtmEnemy1, sword.hit_enemy);
 		
 		// Add collision logic
 		FlxG.collide(blockGroup, player);
 		FlxG.collide(player, sentry1);
 		FlxG.collide(map, dtmEnemy1);
 		FlxG.collide(map, bullets);
 		FlxG.collide(blockGroup, bullets);
+		if(player.equipped_item != sword){
+			FlxG.collide(map, sword);
+			FlxG.collide(blockGroup, sword);
```

Comment:
Correct. I only added one sword to the level, but if there were more, they would be put into a Group and checked the collision there.